Packs/AzureWAF/Integrations/AzureWAF/README.md

```diff
@@ -42,18 +42,25 @@ At end of the process, you will see a message that you logged in successfully.
 ## Configure AzureWAF on Cortex XSOAR
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
-2. Search for AzureWAF.
+2. Search for Azure Web Application Firewall.
 3. Click **Add instance** to create and configure a new integration instance.
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | app_id | App ID | False |
+    | App ID |  | False |
+    | Default Subscription ID |  | True |
+    | Default Resource Group Name |  | True |
+    | Authentication Type | Type of authentication - can be Authorization Code Flow \(recommended\), Device Code Flow, or Azure Managed Identities. | True |
```

Comment:
```suggestion
    | Authentication Type | Type of authentication. Can be Authorization Code Flow \(recommended\), Device Code Flow, or Azure Managed Identities. | True |
```

Reply:
@thefrieddan1 pay attention to this comment of Shirley